README.md

```diff
@@ -1,80 +1,90 @@
-
-### Running on the cluster
-1. Install Instances of Custom Resources:
-
-```sh
-kubectl apply -f config/samples/
-```
-
-2. Build and push your image to the location specified by `IMG`:
-	
-```sh
-make docker-build docker-push IMG=<some-registry>/multi-cluster-traffic-controller:tag
-```
+## Description:
+When deploying the multi cluster traffic controller using the make targets the following will be created: 
+* Kind cluster(s)
+* Gateway API CRDs in the control plane cluster
+* Ingress controller
+* Cert manager
+* External DNS
+* Webhook and webhook config
+* ARGO CD instance
+* K8s Dashboard
+* Lets encrypt certs
 	
-3. Deploy the controller to the cluster with the image specified by `IMG`:
-
-```sh
-make deploy IMG=<some-registry>/multi-cluster-traffic-controller:tag
-```
-
-### Uninstall CRDs
-To delete the CRDs from the cluster:
-
-```sh
-make uninstall
-```
-
-### Undeploy controller
-UnDeploy the controller to the cluster:
-
-```sh
-make undeploy
-```
-
-## Contributing
-// TODO(user): Add detailed information on how you would like others to contribute to this project
-
-### How it works
-This project aims to follow the Kubernetes [Operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
-
-It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controller/) 
-which provides a reconcile function responsible for synchronizing resources untile the desired state is reached on the cluster 
-
-### Test It Out
-1. Install the CRDs into the cluster:
-
-```sh
-make install
-```
-
-2. Run your controller (this will run in the foreground, so switch to a new terminal if you want to leave it running):
-
-```sh
-make run
-```
-
-**NOTE:** You can also run this in one step by running: `make install run`
-
-### Modifying the API definitions
-If you are editing the API definitions, generate the manifests such as CRs or CRDs using:
 
-```sh
-make manifests
-```
 
-**NOTE:** Run `make --help` for more information on all potential `make` targets
+## Prerequisites:
+* AWS
+* Kind 
+    * `make kind`
+* yq 
+    * `make yq`
+
+### 1. Running the operator in the cluster:
+
+
+1. Setup your local environment 
+    ```sh
+    make local_setup MCTC_WORKLOAD_CLUSTERS_COUNT=<NUMBER_WORKLOAD_CLUSTER>
+    ```
+
+1. Update the manager to contain the necessary environment variable and update the image policy:
+    ```
+    containers:
+        - command:
+            - /manager
+            args:
+            - --leader-elect
+            image: controller:latest
+            imagePullPolicy: Never
+            env:
+            - name: AWS_ACCESS_KEY_ID
+            value: <AWS_ACCESS_KEY_ID>
+            - name: AWS_SECRET_ACCESS_KEY
+            value: <AWS_SECRET_ACCESS_KEY>
+            - name: AWS_DNS_PUBLIC_ZONE_ID
+            value: <AWS_DNS_PUBLIC_ZONE_ID>
+            - name: ZONE_ROOT_DOMAIN
+            value: <ZONE_ROOT_DOMAIN>
+    ```
```

Comment:
I don;t think we should have to do this. @sergioifg94 Had a PR to add in a local deploy which would have these already set. If this is what you have to do now, fair enough, but i;d suggest we try and get rid of it asap and have a nicer way of deploying it locally.

Reply:
Im hoping Sergios pr gets merged before we tag phase 1 but right now it's not so I just wrote it as is. if it does get merged soon ill update it